ijkmedia/ijkplayer/android/pipeline/ffpipenode_android_mediacodec_vdec.c

```diff
@@ -2060,7 +2060,7 @@ IJKFF_Pipenode *ffpipenode_create_video_decoder_from_android_mediacodec(FFPlayer
     }
 
     jsurface = ffpipeline_get_surface_as_global_ref(env, pipeline);
-    ret = reconfigure_codec_l(env, node, jsurface);
+    ret = configure_codec_l(env, node, jsurface);
```

Comment:
I have no context to see if this makes the trick or not.

Reply:
description updated

Reply:
I would add here a comment to mention original implementation uses reconfigure. If later we see we can improve something here is good to have such information.

Reply:
Done